### DISCUSSION TITLE
Buttons for playback controls?

### DISCUSSION THREAD

**Discussion by ouvoun** [codetheweb/muse#960] [Ideas]
Hey! Great bot. 

It would be pretty slick if there were buttons added to the bot's messages that added transport controls, instead of relying on slash commands. Is something like that aligned with this project's goals?

**Comment by codetheweb** [comment]
I like the idea, the only complication is whether the embed automatically updates (assuming it's showing the currently playing track or similar). At one point I was thinking of adding an interactive/updating embed, but after community members tried it out most didn't like it because it was unclear if any given embed had the most current data.

**Reply by ouvoun** [reply]
Yeah, that's a very valid complication. The way I've seen this done before is to always pin the transport controls to the latest embed.

**Reply by imjuniper** [reply]
Maybe only having one embed at a time could work too?
